### DISCUSSION TITLE
启动器更新失败：安装更新失败？？？

### DISCUSSION THREAD

**Discussion by satorinoha** [Meloong-Git/PCL#1519] [提问] [answered]
启动器更新失败:安装更新失败
Caused By:无法成功完成操作，因为文件包含病毒或潜在的垃圾软件。
在PCL.ModMain.UpdateRestart(Boolean TriggerRestartAndByEnd)
在PCL.ModMain._Closure$__112-0._Lambda$__1()
在PCL.ModLoader.LoaderTask'2._Closure$__12-0._Lambda$__0()
错误类型:System.ComponentModel.Win32Exception

**Comment by MoYuan-CN** [comment]
此问题可前往 Issue 界面进行反馈

**Comment by WForst-Breeze** [comment, marked as answer]
可以参考 #904 中龙猫提及的解决方案

**Reply by satorinoha** [reply]
谢谢
